importer/RuRu.py

```diff
@@ -0,0 +1,107 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class RuRu(Monument):
+
+    def set_adm_location(self):
+        """
+        Set the administrative location.
+
+        Preferably try to resolve the 'district' (район/округ,
+        the field contains different types so that's why we're
+        matching it against the whole big list of administrative
+        divisions of different kinds).
+        If that doesn't work, use the higher level субъект.
+        """
+        level_three = self.data_files["administrative"]
+        level_two = self.data_files["subyekty"]
+        district_match = [x for x in level_three if x[
+            "itemLabel"] == self.district]
+        if len(district_match) == 1:
+            self.add_statement("located_adm", district_match[0]["item"])
+        else:
+            sub_match = [x for x in level_two if x[
+                "value"].lower() == self.region_iso]
+            if len(sub_match) == 1:
+                self.add_statement("located_adm", sub_match[0]["item"])
+            else:
+                self.add_to_report("district", self.district, "located_adm")
+
+    def set_location(self):
```

Comment:
With [T174934](https://phabricator.wikimedia.org/T174934) it should be possible to get to the Qid directly from `city_id` if that has been added on Wikivoyage. The current structure can be used as a fallback.